nixos/modules/services/development/zammad.nix

```diff
@@ -0,0 +1,217 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.zammad;
+  serviceConfig = {
+    Type = "simple";
+    Restart = "always";
+
+    User = "zammad";
+    Group = "zammad";
+    PrivateTmp = true;
+    StateDirectory = builtins.baseNameOf cfg.dataDir;
```

Comment:
this may not work as inteded if `dataDir` is changed to something outside of `/var/lib`. other services don't even make this configurable and use the service name instead.